content/github/managing-subscriptions-and-notifications-on-github/managing-notifications-from-your-inbox.md

```diff
@@ -81,6 +81,7 @@ Custom filters do not currently support:
   - Distinguishing between the `is:issue`, `is:pr`, and `is:pull-request` query filters. These queries will return both issues and pull requests.
   - Creating more than 15 custom filters.
   - Changing the default filters or their order.
+  - Search [exclusion](https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/understanding-the-search-syntax#exclude-certain-results) using `NOT` or `-QUALIFIER`.
```

Comment:
Many thanks for adding this.

To accommodate page versioning, we leave off the first part of URLs in links to other GitHub docs pages: 
```suggestion
  - Search [exclusion](/github/searching-for-information-on-github/understanding-the-search-syntax#exclude-certain-results) using `NOT` or `-QUALIFIER`.
```